Makefile

```
@@ -13,4 +13,5 @@ endif
 
 
 sourcecode: draft-ietf-jsonpath-base.xml
+	rm -rf sourcecode.ba?
```

Why does this not remove `sourcecode.bak`?

---

> Why does this not remove `sourcecode.bak`?

It does.  kramdown-rfc-extract-sourcecode then creates a new backup of the existing sourcecode directory.
A bit belt and suspenders, but I wanted to make sure the tool doesn't delete user information.
(We could remove the existing sourcecode directory as well, but then recovery is a bit harder if the sourcecode extraction fails for some reason.)

---

Sorry, I didn't express myself very well. Why doesn't the code say the following?
```suggestion
	rm -rf sourcecode.bak
```
Are there some other files/directories matching `sourcecode.ba?` that also need removing?

---

Because if the backup mechanism can't create a .bak, it will create .bal, .bam etc.
So if you did an extract-sourcecode manually a few times, you might have all these dropping.
.ba? takes care of all ~15 of them.